### PR TITLE
Adding the task 1 files 1-hbnb.py, 1-hbnb.js and 1-hbnb.html

### PR DESCRIPTION
**Pull request by KevinCastroP**

### CODE REVIEW

File: web_dynamic/static/scripts/1-hbnb.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+Your script must be executed only when DOM is loaded
+You must use jQuery
+Listen for changes on each INPUT checkbox tag
+*/
+
+$(document).ready(function() {
+    console.log("ready!");
```
Comment (oscargtoro) [inline web_dynamic/static/scripts/1-hbnb.js]: clean unnecessary console.log

File: web_dynamic/static/styles/6-filters.css [hunk 2]
```diff
@@ -101,3 +101,8 @@ button:hover {
 .locations:hover .popover {
     visibility: visible;
 }
+
+.amenities > .popover > li {
+    margin-left: 50px;
+    margin-bottom: 10px;
+}
```
Comment (oscargtoro) [inline web_dynamic/static/styles/6-filters.css]: File not ending with a new line

File: web_dynamic/static/scripts/1-hbnb.js (hunk 1 above)
Comment (KevinCastroP) [reply web_dynamic/static/scripts/1-hbnb.js]: ok, I will fix it

File: web_dynamic/static/styles/6-filters.css (hunk 2 above)
Comment (KevinCastroP) [reply web_dynamic/static/styles/6-filters.css]: ok, I'm going to fix it

**Comment by oscargomez87** [comment]
Last requirement not being met.

> Listen for changes on each INPUT checkbox tag:
> 
> - if the checkbox is checked, you must store the Amenity ID in a variable (dictionary or list)
> 
> - if the checkbox is unchecked, you must remove the Amenity ID from the variable
> 
> - update the H4 tag inside the DIV Amenities with the list of Amenities checked

**Review by oscargtoro — COMMENTED** [review]
Last requirement not met.
> Listen for changes on each INPUT checkbox tag:
> - if the checkbox is checked, you must store the Amenity ID in a variable (dictionary or list)
> - if the checkbox is unchecked, you must remove the Amenity ID from the variable
> - update the H4 tag inside the DIV Amenities with the list of Amenities checked

**Review by oscargtoro — COMMENTED** [review]
Once you select more elements the h4 tag deforms and spills out of the container

**Review by oscargtoro — APPROVED** [review]
Everything working as per requirements

**Pull request merged**